chart/values.schema.json

```diff
@@ -857,6 +857,15 @@
                         "description": "Type **as string** of secret E.G. Opaque, kubernetes.io/dockerconfigjson, etc.",
                         "type": "string"
                     },
+                    "labels": {
+                        "description": "Labels for the secret",
+                        "type": "object",
+                        "additionalProperties": {
+                            "description": "Label key",
+                            "default": null,
```

Comment:
Shouldn't this default be set up a level, as the default for `labels`?

Reply:
Moved up a level